testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/microprofile/opentracing/ResourceNotTracedTestCase.java

```diff
@@ -30,6 +53,22 @@
     @ArquillianResource
     private URL url;
 
+    /**
+     * Permissions required in case security manager is enabled.
+     */
+    private static final String PERMISSIONS = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
```

Comment:
There is a [utility](https://github.com/wildfly/wildfly/blob/bf07fdae275a8fd208d483d4670324fd28c2d88f/testsuite/shared/src/main/java/org/jboss/as/test/shared/integration/ejb/security/PermissionUtils.java#L42) for this. Have a look at this [example](https://github.com/wildfly/wildfly/blob/4eaf7e53290b5e6a0c84b78efbafaff7b014b87c/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/batch/chunk/ChunkPartitionTestCase.java#L74-L79).